js/ui/deskletManager.js

```diff
@@ -252,7 +253,7 @@ function _onEnabledDeskletsChanged() {
 
     // Make sure all desklet extensions are loaded.
     // Once loaded, the desklets will add themselves via finishExtensionLoad
-    initEnabledDesklets();
+    initEnabledDesklets().then(updateMouseTracking);
```

Comment:
The definitions array handling is synchronous and is already determined before we start loading desklets, so we don't need to call this through the promise prototype. The promise is only needed if we want to have something run after every desklet is loaded from disk. The cursor movement improves when I call updateMouseTracking before initEnabledDesklets. Maybe a CJS/GLib bug (since the promise itself is implemented as an idle source)?

Reply:
I have no idea why it would have that effect, so I can't really say